erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -89,7 +89,7 @@ def get_columns(filters):
 		]
 
 	columns += [
-		_("Voucher Type") + "::120", _("Voucher No") + ":Dynamic Link/"+_("Voucher Type")+":160",
+		"Voucher Type" + "::120", _("Voucher No") + ":Dynamic Link/Voucher Type:160",
```

Comment:
don't you think that column label "Voucher Type" will not be translatable? I think you should leave it as _("Voucher Type")
I suppose you just don't need it in "Dynamic Link/Voucher Type:160"
btw, how about line 95, does everything work?
and sorry, I use ERPNext: v8.0.36 and I cannot replicate this problem